index.js

```diff
@@ -13,15 +13,16 @@ var DEFAULTS = {
 	IE: false
 }
 
+var LAST_KNOWN_EDGEHTML_VERSION = 18
 var EDGEHTML_VS_EDGE_VERSIONS = {
 	12: 0.1,
 	13: 21,
 	14: 31,
 	15: 39,
 	16: 41,
-	17: 42,
-	18: 44
+	17: 42
 }
+EDGEHTML_VS_EDGE_VERSIONS[LAST_KNOWN_EDGEHTML_VERSION] = 44
```

Comment:
Why not get the most recent entry from the Object?

Reply:
Not exactly sure what you mean here, take a look at my earlier comments though. Basically I couldn't figure out how to get last entry from the object in a way <IE9.
* `versions[versions.length - 1]` doesn't work, returns undefined.
* Object.keys(versions) into an array and then access like above works but it's modern.
* Object.getOwnPropertyNames(versions) into an array and then accessed like above works but is IE9+

The only thing I could think of was a helper function with `for prop in versions`, checking for `hasOwnProperty`, assigning to a local var and returning the last one after the loop. But at that point it just seemed easier to abstract the last known version out and build the map with it too.

Open to any ideas you've got though!

Reply:
I'd get a polyfill for `Object.keys()`

Reply:
There's one on MDN, but it doesn't specify how far back it supports...

Reply:
I think it'll go back a ways, but I also don't think we'll ever have any more edge versions, so maybe it's fine as we'll never have to change that version map again. We'll have to see what Edge on Chromium looks like.

Reply:
Are you happy to not worry about above and we can finish this off?